source/Octopus.Cli/Commands/Package/DeletePackage.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Repositories;
+using Octopus.Cli.Util;
+using Octopus.Client;
+using Octopus.Client.Model;
+using Octopus.CommandLine;
+using Octopus.CommandLine.Commands;
+using Octopus.Versioning.Octopus;
+namespace Octopus.Cli.Commands.Releases
+{
+    [Command("delete-package", Description = "Deletes a package from the built-in NuGet repository in an Octopus Server.")]
+    public class DeletePackageCommand : ApiCommand, ISupportFormattedOutput
+    {
+        static readonly OctopusVersionParser OctopusVersionParser = new OctopusVersionParser();
```

Comment:
nit: doesn't appear to be used, can it be removed?

Reply:
Done